modules/dkan/dkan_dataset/dkan_dataset.install

```diff
@@ -129,3 +129,39 @@ function dkan_dataset_update_7003() {
   $vocabulary = taxonomy_vocabulary_machine_name_load('odfe_theme');
   taxonomy_vocabulary_delete($vocabulary->vid);
 }
+
```

Comment:
@janette our standard is to only add update functions to dkan.install since the repo consolidation